packages/gatsby-theme-carbon/src/components/GifPlayer/GifPlayer.js

```diff
@@ -0,0 +1,101 @@
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+import classnames from 'classnames';
+import {
+  PlayOutline24,
+  PlayOutlineFilled24,
+  PauseOutline24,
+  PauseOutlineFilled24,
+} from '@carbon/icons-react';
+import styles from './GifPlayer.module.scss';
+
+const Pause = ({ hovering }) =>
+  hovering ? <PauseOutlineFilled24 /> : <PauseOutline24 />;
+
+const Play = ({ hovering }) =>
+  hovering ? <PlayOutlineFilled24 /> : <PlayOutline24 />;
+
+const ToggleIcon = ({ paused, hovering }) =>
+  paused ? <Play hovering={hovering} /> : <Pause hovering={hovering} />;
+
+const GifPlayer = ({ children, color, className, isInDialog }) => {
+  const [paused, setPaused] = useState(false);
+  const [hovering, setHovering] = useState(false);
+
+  const onClick = () => {
+    setPaused(!paused);
+  };
+
+  const controlsClassNames = classnames({
+    [styles.controls]: true,
+    [styles.dark]: color === 'dark',
+  });
+
+  const containerClassNames = classnames({
+    [styles.container]: true,
+    [className]: className,
+    [styles.gifInDialog]: isInDialog,
+  });
+
+  const staticImageClassNames = classnames({
+    [styles.imgHidden]: true,
+    [styles.imgDisplayed]: paused,
+  });
+
+  const gifClassNames = classnames({
+    [styles.gifDisplayed]: true,
+    [styles.gifHidden]: paused,
+  });
+
+  const childrenArray = React.Children.toArray(children);
+
+  return (
+    <div className={containerClassNames}>
+      <div className={gifClassNames} aria-hidden={paused ? 'true' : false}>
+        {childrenArray[0]}
+      </div>
+      <div
+        className={staticImageClassNames}
+        aria-hidden={paused ? false : 'true'}
+      >
+        {childrenArray[1]}
+      </div>
+      <button
+        type="button"
```

Comment:
```suggestion
```

Reply:
Don't think we need this here. `<button type="button">` as far as I know communicates that the button does nothing when pressed.

[reference](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/button)

Reply:
@dakahn I added this because the gatsby theme eslint rules require a `type` to be specified for buttons, and I wasn't sure if this would be considered `type="submit"` if it's not submitting any data. cc: @vpicone

Reply:
@dakahn @jnm2377 type button is correct buttons default to type submit so you need to define them as type button to avoid awkward form submission behavior. 
Type=“button” is what you’d expect to see here.

Reply:
Reading the link I referenced

_The button **has no default behavior** and does nothing when pressed. It can have client-side scripts associated with the element's events, which are triggered when the events occur._

So I think you're right -- but the only part that's throwing me is the "does nothing when pressed" bit. I'm not sure if that would be communicated to screen reader users who we would want to expect the button in question to open a menu. 

Not a blocker tho!

Reply:
@dakahn not sure if it makes a difference, but when I test this with voice over it says it's a toggle button. Is that not a sufficient indication of what it does? I also updated the button label to say if it's playing or paused.

Reply:
I'm not sure what is the expected voice over but this is what it does right now. 
![voiceover](https://user-images.githubusercontent.com/32556167/73030439-44e47880-3dff-11ea-8c54-e4d43d1577ae.gif)

Reply:
That's great!